crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1320,7 +1355,7 @@ fn visit_node(
         NodeData::Rectangle { vector } => ViewShape::VectorRect {
             path: fill_paths,
             stroke: stroke_paths,
-            corner_radius,
+            corner_radius: corner_radius,
```

Comment:
Wondering if here should use corner_radius.clone() as above?

Reply:
It's not needed here. It's needed above because of how Rust memory management works. Without the clone above, the local `corner_radius` variable will have been moved, or consumed, into the `ViewShape::RoundRect` enum, and there would be a "corner_radius has been moved" compilation error. Since nothing below this uses `corner_radius`, it doesn't need a clone.